src/js/stores/MetronomeStore.js

```diff
@@ -166,10 +166,40 @@ class MetronomeStore extends EventEmitter {
     MetronomeActions.runJob(jobID);
   }
 
-  suspendSchedule(jobID) {
-    let schedule = this.getJob(jobID).schedules[0];
-    schedule.enabled = false;
-    MetronomeActions.suspendSchedule(jobID, schedule);
+  disableSchedule(jobID) {
+    let job = this.getJob(jobID);
+    if (job == null) {
```

Comment:
We need eslint rules for all of this! It's really tedious to give and receive this kind of feedback. Would you mind checking our best practice doc and also have a look, if there is a eslint rule that we could apply?